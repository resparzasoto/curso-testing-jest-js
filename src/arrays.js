const fruits = ['banana', 'cantaloupe', 'watermelon', 'orange', 'lemon', 'pickle'];
const colors = ['blue', 'green', 'red', 'pink', 'yellow'];

export const arrayFruits = () => fruits;
export const arrayColor = () => colors;