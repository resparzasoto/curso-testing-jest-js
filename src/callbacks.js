export const callbackHell = (callback) => {
    callback('Hi JavaScripts!');
};