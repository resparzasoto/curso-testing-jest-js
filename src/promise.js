import Axios from 'axios';

export const getDataFromApi = url => {
    return Axios.get(url)
        .then(({ data }) => {
            return data;
        });
}