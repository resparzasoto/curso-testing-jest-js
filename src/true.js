export const isTrue = () => true;

export const isFalse = () => false;

export const isNull = () => null;

export const isUndefined = () => undefined;