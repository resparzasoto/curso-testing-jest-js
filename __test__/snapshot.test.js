import { getCharacter } from '../src/snapshot';
import rick from '../rick.json';

describe('ItsTimeToSnapshots', () => {
    test('SnapshotOfRick', () => {
        expect(getCharacter(rick)).toMatchSnapshot();
    });
});