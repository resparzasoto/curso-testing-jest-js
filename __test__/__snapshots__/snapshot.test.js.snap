// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItsTimeToSnapshots SnapshotOfRick 1`] = `
Object {
  "gender": "Male",
  "id": 1,
  "name": "Rick Sanchez",
  "species": "Human",
  "status": "Alive",
}
`;
